.github/workflows/Image-build-NJDG-Transformer.yaml

```
@@ -0,0 +1,113 @@
+      - 'backend/njdg-transformer/**'
+  workflow_dispatch:
+  workflow_call:
+permissions:
+  security-events: write
+  contents: read
+jobs:
+  docker_image-build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)